week3/Homework/classmod.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  </head>
+  <body>
+    <button>Sample</button>
+<p></p>
+    <tr>
+      <th>Name</th>
+      <th>Homework</th>
+      <th>Quiz</th>
+      <th>Exam</th>
+      <th>GPA</th>
+    </tr>
+
+    <script>
+var students = 15;
+var homework = 5;
+var exam = 2;
+
+$("BUTTON").click(() =>{
+  document.write("<table><tr> <th>Name</th> <th>Homework</th> <th>Exam</th> <th>Average</th> <th>GPA</th> </tr>");
+  var y = 0;
+
+while (students > y){
+var array = [];
+var x = 0;
+var rand = 0;
+
+var respons;
+  $.get("https://randomuser.me/api/",
+    (response) => {
+      JSON.stringify(response);
+
+       var respons = response["results"][0]["name"]["title"];
```

Comment:
with a better name on the left of the assignment, this line would read better, and you wouldn't need the `var name = respons` below